tools/kafka-versions-tools.sh

```diff
@@ -38,6 +38,65 @@ function get_kafka_versions {
 
 }
 
+function get_kafka_source_urls {
+
```

Comment:
Do we really need all the empty lines everywhere? I do not think it helps anything.

Reply:
Well I think it makes it easier to read the code. But this is just a matter of taste, so I will take them out.